### DISCUSSION TITLE
Screen recording under qtile

### DISCUSSION THREAD

**Discussion by singpolyma** [qtile/qtile#2920] [Q&A] [answered]
I launched my normal obs stack today after switching from dwm to qtile and found that obs could only see the contents on windows that were on the currently-selected group.  Under dwm it can see any window on any tag.  Not sure what the difference is here.

**Comment by m-col** [comment, marked as answer]
> Under dwm it can see any window on any tag. 

Is this every window, including those that are completely hidden? How do tags correspond to groups?

**Reply by m-col** [reply]
From what I can tell, tags are also workspaces like groups, but windows can be on multiple tags at the same time. Is that correct?

**Reply by singpolyma** [reply]
Yes, including windows that are completely hidden (only on tags not currently being shown).  And yes, tags are pretty analogous to groups, dwm allows you to put a window onto multiple tags or to view multiple tags at once, but if you don't use either of those they are the same as groups.

**Reply by m-col** [reply]
Looks like dwm doesn't actually unmap windows, it just shifts them really far to the left off screen. This is why you can still select "hidden" windows. I don't think this is something we will consider doing as this comes with its own costs e.g. unmapped windows do not waste time rendering their contents whereas mapped windows far off screen will continue to do so.

**Reply by singpolyma** [reply]
Good to know, thanks.  It's no problem to switch back to dwm for my streaming setup as I don't do that daily, I mostly just wanted to know what the difference was.  Thanks.

**Comment by Quicksilver151** [comment]
im facing this issue as well
is there a way to prevent qtile from unmapping windows?

**Reply by nopium** [reply]
I would really like to now this as well. Is it even possible to record another "tile" from obs except the full screen?
